addons/mail/static/src/models/message/message.js

```diff
@@ -112,7 +112,14 @@ function factory(dependencies) {
                 }
             }
             if ('partner_ids' in data && this.messaging.currentPartner) {
-                data2.isCurrentPartnerMentioned = data.partner_ids.includes(this.messaging.currentPartner.id);
+                data2.recipients = insertAndReplace(data.partner_ids.map(partner_id =>
+                    this.messaging.models['mail.partner'].convertData({id: partner_id})
```

Comment:
lint: should have space inside inline object `{ id: ... }`

Reply:
Also the `convertData` part is useless when its just `id` that is given, this could be simplified a bit.